include/yboard.h

```diff
@@ -102,16 +103,66 @@ class YBoardV3 {
+     * played, play_notes is called, or the song finishes. After this function is called, the
+     * loop_speaker function must be called often to playback the sound on the speaker.
+     */
+    bool play_song_from_sd_background(const char *filename);
+
     /*
      *  This function sets the speaker volume. The volume is an integer between 0
-     * and 100. A volume of 0 is off, and a volume of 100 is full volume.
+     * and 10. A volume of 0 is off, and a volume of 10 is full volume.
+     */
+    void set_sd_song_volume(uint8_t volume);
+
+    /* Plays the specified sequence of notes. The function will return once the notes
+     * have finished playing.
+     *
+     * A–G	                Specifies a note that will be played.
+     * R                    Specifies a rest (no sound for the duration of the note).
+     * + or # after a note  Raises the preceding note one half-step (sharp).
+     * - after a note	      Lowers the preceding note one half-step.
+     * > after a note	      Plays the note one octave higher (multiple >’s can be used, eg: C>>)
+     * < after a note	      Plays the note one octave lower (multiple <’s can be used, eg: C<<)
+     * 1–2000 after a note	Determines the duration of the preceding note. For example,
+     *                      C16 specifies C played as a sixteenth note, B1 is B played as a whole
+     *                      note. If no duration is specified, the note is played as a quarter note.
+     * O followed by a #    Changes the octave. Valid range is 4-7. Default is 5.
+     * T followed by a #    Changes the tempo. Valid range is 40-240. Default is 120.
+     * V followed by a #    Changes the volume.  Valid range is 1-10. Default is 5.
+     * !                    Resets octave, tempo, and volume to default values.
+     * spaces               Spaces can be placed between notes or commands for readability,
+     *                      but not within a note or command (eg: "C4# D4" is valid, "C 4 # D 4" is
+     *                      not. "T120 A B C" is valid, "T 120 A B C" is not).
+     */
+    void play_notes(std::string notes);
+
+    /* This is similar to the function above, except that it will start playing the notes
+     * in the background and return immediately. The notes will continue to play in the
+     * background until they are stopped with the stop_audio function, a WAVE file is played,
+     * or the notes finish. If you call this function again before the notes finish, the
+     * the new notes will be appended to the end of the current notes.  This allows you to
+     * call this function multiple times to build up multiple sequences of notes to play.
+     * After this function is called, the loop_speaker function must be called often to
+     * playback the sound on the speaker.
+     */
+    void play_notes_background(std::string notes);
+
+    /*
+     * This function stops the audio from playing (either a song or a sequence of notes)
+     */
+    void stop_audio();
```

Comment:
Just curious, can someone call `stop_audio` and then play a sound from the SD right away?

Reply:
Yes, that should work fine.  I'll double check.